src/main/java/pl/touk/parking/web/DriverDto.java

```diff
@@ -0,0 +1,26 @@
+package pl.touk.parking.web;
+
+import pl.touk.parking.domain.Driver;
+
+import javax.validation.constraints.NotNull;
+
+public class DriverDto {
+
+    @NotNull(message = "vehicleNumber is required")
+    String vehicleNumber;
+
+    @NotNull(message = "driverType is required")
+    DriverType driverType;
+
+    public Driver create() {
+        return new Driver(vehicleNumber, driverType.getPaymentPolicy());
```

Comment:
czy taka zależność jest OK?

Reply:
Chodzi o to, że DTO wie jak utworzyć obiekt domenowy? Zawiera wszystkie dane potrzebne do tego a, jesli zmieni się `Driver` to zmiana dotyczyć będzie tego miejsca a nie wszystkich klientów tej klasy. Według mnie OK.